spec/index.html

```diff
@@ -748,6 +773,31 @@ <h2>Literals</h2>
       same literal <a>RDF terms</a> and are not
       <a>term-equal</a> because their
       <a>lexical form</a> differs.</p>
+
+    <section id="section-text-direction" class="informative">
+      <h3>Initial Text Direction</h3>
+
+      <p>The <a>base direction</a> of a <a>directional language-tagged string</a>
+        provides a means of establishing the initial direction of text,
+        including text which is mixture of right-to-left and left-to-right scripts.
+        The [[[?UAX9]]] [[?UAX9]] provides support for automatically rendering
+        a sequence of characters in logical order,
+        so that they are visually ordered as expected.
+        However, this is not sufficient to correctly render bidirectional text.</p>
+
+      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" (for Hebrew) may be displayed
+        incorrectly unless the <a>base direction</a> of `rtl` (for right-to-left)
+        is also provided.
+        When this is provided to a user agent,
+        for example using the HTML `dir` attribute,
+        it would be correctly presented as <br/><span lang="he" dir="rtl">פעילות הבינאום, W3C</span>.</p>
```

Comment:
```suggestion
        it would be correctly presented as follows: <div lang="he" dir="rtl">פעילות הבינאום, W3C</div></p>
```
with a `<div>`, the text is actually justified to the right, which makes it more obvious that it is RTL (and probably more correct).